cinema_project/base_app/views.py

```diff
@@ -66,12 +65,7 @@ def bookings_page(request):
 
 
 def city_filtered_page(request, selected_city):
-    schedules_by_city = Schedule.objects.filter(
-        schedule_time__range=[today(), next_days(7)],
-        hall__cinema__city__iexact=selected_city
-    )
-    cinema_names = list(set(schedule.hall.cinema.name for schedule in schedules_by_city))
-    cinema_names.sort()
+    cinema_names = list(Cinema.objects.filter(city=selected_city).values_list('name', flat=True).order_by('name'))
```

Comment:
Not sure if this is on purpose, but the schedule_time__range filter has been left out. This can just be included along the city=selected_city filter.

Reply:
It was on purpose, thought I only needed all the cinemas, but this way, I can display only cinemas that have a movie scheduled, which fixes a bug I had, which is 2 for one, great!
By applying this change I also learned how to use values_list to get whichever value I want, Thanks!